app/controllers/users_controller.rb

```diff
@@ -1,3 +1,30 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find_by id: params[:id]
+  end
+
+  def index
+    @user = User.all
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t ".success"
+      redirect_to user_path @user
+    else
+      render "new"
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:name, :email, :password,
```

Comment:
bỏ cặp dấu ( ) ở permit
```suggestion
    params.require(:user).permit :name, :email, :password,
```